test/Project.toml

```diff
@@ -13,4 +13,4 @@ UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
 ConstructionBase = "1.5.1"
-ObjectFile = "0.3.6"
+ObjectFile = "0.3.6, 0.4"
```

Comment:
Does this really work with both ObjectFile 0.3.6 _and_ 0.4?  If I try to iterate over a handle with ObjectFile 0.3.6 I get
```
julia> readmeta("libjulia.dylib") do ohs
           foreach(ohs) do oh
               symbols = symbol_name.(Symbols(oh))
           end
       end
ERROR: MethodError: no method matching iterate(::MachOHandle{IOStream})
```
At this point we can just set compatibility with 0.4 and move on.

Reply:
Good catch. I somehow thought that scalars would automatically be an iterator of length 1, but of course that's only true for numbers.